drivers/kw41zrf/kw41zrf.c

```diff
@@ -39,23 +38,24 @@
 static void kw41zrf_set_address(kw41zrf_t *dev)
 {
     DEBUG("[kw41zrf] Set MAC address\n");
-    eui64_t addr_long;
-    network_uint16_t addr_short;
 
     /* get unique IDs to use as hardware addresses */
-    luid_get_eui64(&addr_long);
-    luid_get_short(&addr_short);
+    netdev_ieee802154_setup(&dev->netdev);
```

Comment:
This should be moved to the setup part.

E.g. assume someone would use the `ifconfig` shell command to manually change the L2 address and afterwards use again `ifconfig` to reset the device. With this, the old L2 address would be restored - which is surprising. Just moving this line to the setup or init code would fix it.

Reply:
I wasn't sure if 'reset' should mean: "Restore default values" or just reset the radio.

I'll move it then.